config/spec_helper.rb

```diff
@@ -0,0 +1,102 @@
+require 'simplecov'
```

Comment:
No backtics, hooray! This is one of those frustrating moments that will ensure we never make the same mistake twice.